### ISSUE TITLE
Does 2 same adapter work with wifiphisher?

### ISSUE THREAD

**Issue by lwj88** [wifiphisher/wifiphisher#638]
So I am going to buy TL-WN722N because it is more cheap compare to other adapter which is compatible with Kali Linux. I already owned one so I want to ask does 2 same adapter work with wifiphisher? Does TL-WN722N support AP mode (I know it support monitor mode)? Please help, thank you.

**Comment by JetseVerschuren** [comment]
Yes, but the TL-WN722N supports only 7 clients at the same moment.
[source](https://www.raspberrypi.org/forums/viewtopic.php?f=36&t=76604)

**Comment by anakin1028** [comment]
You can use the command `iw dev phy phy0 info` to see the supported mode for your adapter card. The new model of `TL-WN722N` uses the `Realtek` chipset instead of `Atheros` which may not support the AP or Monitor mode.  You can have a look on #558

**Comment by lwj88** [comment]
Thanks for the replies.@anakin1028 I think I will buy other adapter which is recommended by my friend (AP Mode supported) because I will buy through online and because of this I cannot cannot confirm that TL-WN722N chipset I bought is Realtek instead of Atheros.
